plugin/color.py

```diff
@@ -74,7 +73,7 @@ def send_color_request(self) -> None:
         if is_transient_view(self.view):
             return
 
-        client = client_from_session(session_for_view(self.view, 'colorProvider'))
+        client = client_from_session(session_for_view(self.view, 'colorProvider', self.view.sel()[0].b))
```

Comment:
Is this a correct change?
I would think that the `documentColor` request should be sent regardless of where the cursor is.

Reply:
This always happened behind the curtains. I just made it explicit. Read this carefully:

https://github.com/sublimelsp/LSP/blob/10c4004be55dd54ff61473093f03e2a90ca867ca/plugin/core/configurations.py#L15

Reply:
Oh, OK, I see now.

Previous logic used `region.begin()` rather than `region.b` though. I'm not sure which one would be more correct to honest.

Reply:
`region.b` is better because that's the cursor position. I made the `point` arg optional.

Reply:
> region.b is better because that's the cursor position

I always wondered what the exact cursor position is. And now I know